examples/gno.land/r/stefann/home/home_test.gno

```diff
@@ -0,0 +1,280 @@
+package home
+
+import (
+	"std"
+	"strings"
+	"testing"
+
+	"gno.land/p/demo/avl"
+)
+
+func TestUpdatePFP(t *testing.T) {
+	std.TestSetOrigCaller(std.Address("g1sd5ezmxt4rwpy52u6wl3l3y085n8x0p6nllxm8"))
```

Comment:
Thanks for pointing that out, I missed it in the docs. It might be helpful to include a reference in the `how-to-guides/testing-gno` or `reference/stdlibs/std/testing` sections to encourage newcomers to use it.

Anyway, I’ve updated it only in functions that don’t involve an owner check, as I can’t use `TestAddress()` to provide the specific owner address, not sure if there's a better way? 73c9b7b927483a03bdb87c1f6bbea33850f8a2f9